authors.tex

```diff
@@ -1,6 +1,7 @@
-\author[uiuc]{Firstname M. Lastname}
-\author[uiuc]{John S. Doe}
-\author[uiuc]{Jane S. Doe\corref{corrauthor}}
-%% If unsure, google "corresponding author" for more info
 \cortext[corrauthor]{Corresponding Author}
-\ead{jsdoe@gmail.com}
+\author[uiuc]{Amanda M. Bachmann}
+\ead{amandab7@illinois.edu}
+\author[uiuc]{Kathryn D. Huff\corref{corrauthor}}
+\ead{kdhuff@illinois.edu}
+\author[uiuc, ncsa]{Madicken Munk}
```

Comment:
I swear you're going to hate this, but

```suggestion
\author[uiuc,ncsa]{Madicken Munk}
```

Reply:
Oh my gosh. You're right; I do hate this.